qcodes/instrument_drivers/QuTech/S5i.py

```diff
@@ -20,34 +20,88 @@ class S5i(Instrument):
             individual modules.
 
         module (int): module number as set on the hardware.
+
+    Parameters:
+        enabled: switched output on/off
+        frequency: Frequency on Hz, range 40 MHz - 4 GHz, stepsize 1 MHz
+        freq_fine: Frequency on Hz, range 40 MHz - 4 GHz, stepsize 10 kHz.
+        Difference between frequency and freq_fine unclear!?
+        power: power in dBm, range -20 to +15 dBm
+
     """
 
     def __init__(self, name, spi_rack, module, **kwargs):
         super().__init__(name, **kwargs)
 
-        self.s5i = S5i_module(spi_rack, module)
-
-        self.add_parameter('use_external_reference',
-                           label='Use external reference',
-                           get_cmd=self._use_external_reference,
-                           set_cmd=self.s5i.use_external_reference,
+        self.s5i = S5i_module(spi_rack, module, frequency=41e6,
+                              enable_output=1, output_level=0)
+        # Put low freq as it is unlikely it will be resonant with something.
+        self.add_parameter('enabled',
+                           label='RF_on/off',
+                           set_cmd=self.s5i.enable_output_soft,
                            vals=Bool())
 
-        self.add_parameter('frequency_stepsize',
```

Comment:
if this parameter is removed (actually, is it removed because there is not need for setting the step size separately?), shall the `_get_stepsize` method also be removed?